zulipterminal/ui_tools/views.py

```diff
@@ -293,13 +293,17 @@ def __init__(self) -> None:
 class StreamsView(urwid.Frame):
     def __init__(self, streams_btn_list: List[Any], view: Any) -> None:
         self.view = view
-        self.log = urwid.SimpleFocusListWalker(streams_btn_list)
+        # Create Stream List Box
         self.streams_btn_list = streams_btn_list
-        self.focus_index_before_search = 0
+        self.log = urwid.SimpleFocusListWalker(streams_btn_list)
         list_box = urwid.ListBox(self.log)
+        # Create Stream Search Box
         self.stream_search_box = PanelSearchBox(self,
                                                 'SEARCH_STREAMS',
                                                 self.update_streams)
+        self.focus_index_before_search = 0
+        self.in_search_mode = False
```

Comment:
We use `is_in` prefix for booleans that track whether an entity _is_ currently _in_ some state. For instance, see `is_in_typeahed_mode` in our source.

Reply:
:+1: I'll change it to `is_in_search_mode`